Bottom: a34d015d2c6a48e033d3d118f6777a0c83f794af
Top:    df3d669607c5eb1094d6dabf81338bd126c32aee
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 16:36:56 +0000

Refresh of add-a-hacky-name-mangling-thin

---

diff --git a/src/create.h b/src/create.h
index 3397091..97616fd 100644
--- a/src/create.h
+++ b/src/create.h
@@ -43,27 +43,27 @@ PCBType * CreateNewPCB (bool);
 /* Called after PCB->Data->LayerN is set.  Returns zero if no errors,
    else nonzero.  */
 int CreateNewPCBPost (PCBType *, int /* set defaults */);
-PinType *CreateNewVia (DataType *, Coord, Coord, Coord, Coord, Coord, Coord, char *, FlagType);
-LineType *CreateDrawnLineOnLayer (LayerType *, Coord, Coord, Coord, Coord, Coord, Coord, FlagType);
-LineType *CreateNewLineOnLayer (LayerType *, Coord, Coord, Coord, Coord, Coord, Coord, FlagType);
-RatType *CreateNewRat (DataType *, Coord, Coord, Coord, Coord, Cardinal, Cardinal, Coord, FlagType);
-ArcType *CreateNewArcOnLayer (LayerType *, Coord, Coord, Coord, Coord, Angle, Angle, Coord, Coord, FlagType);
-PolygonType *CreateNewPolygonFromRectangle (LayerType *, Coord, Coord, Coord, Coord, FlagType);
-TextType *CreateNewText (LayerType *, FontType *, Coord, Coord, unsigned, int, char *, FlagType);
-PolygonType *CreateNewPolygon (LayerType *, FlagType);
-PointType *CreateNewPointInPolygon (PolygonType *, Coord, Coord);
-PolygonType *CreateNewHoleInPolygon (PolygonType *polygon);
+PinType * CreateNewVia (DataType *, Coord, Coord, Coord, Coord, Coord, Coord, char *, FlagType);
+LineType * CreateDrawnLineOnLayer (LayerType *, Coord, Coord, Coord, Coord, Coord, Coord, FlagType);
+LineType * CreateNewLineOnLayer (LayerType *, Coord, Coord, Coord, Coord, Coord, Coord, FlagType);
+RatType * CreateNewRat (DataType *, Coord, Coord, Coord, Coord, Cardinal, Cardinal, Coord, FlagType);
+ArcType * CreateNewArcOnLayer (LayerType *, Coord, Coord, Coord, Coord, Angle, Angle, Coord, Coord, FlagType);
+PolygonType * CreateNewPolygonFromRectangle (LayerType *, Coord, Coord, Coord, Coord, FlagType);
+TextType * CreateNewText (LayerType *, FontType *, Coord, Coord, unsigned, int, char *, FlagType);
+PolygonType * CreateNewPolygon (LayerType *, FlagType);
+PointType * CreateNewPointInPolygon (PolygonType *, Coord, Coord);
+PolygonType * CreateNewHoleInPolygon (PolygonType *polygon);
 void RefdesMapInit (void);
-ElementType *CreateNewElement (DataType *, ElementType *, FontType *, FlagType, char *, char *, char *, Coord, Coord, BYTE, int, FlagType, bool);
-LineType *CreateNewLineInElement (ElementType *, Coord, Coord, Coord, Coord, Coord);
-ArcType *CreateNewArcInElement (ElementType *, Coord, Coord, Coord, Coord, Angle, Angle, Coord);
-PinType *CreateNewPin (ElementType *, Coord, Coord, Coord, Coord, Coord, Coord, char *, char *, FlagType);
-PadType *CreateNewPad (ElementType *, Coord, Coord, Coord, Coord, Coord, Coord, Coord, char *, char *, FlagType);
-LineType *CreateNewLineInSymbol (SymbolType *, Coord, Coord, Coord, Coord, Coord);
+ElementType * CreateNewElement (DataType *, ElementType *, FontType *, FlagType, char *, char *, char *, Coord, Coord, BYTE, int, FlagType, bool);
+LineType * CreateNewLineInElement (ElementType *, Coord, Coord, Coord, Coord, Coord);
+ArcType * CreateNewArcInElement (ElementType *, Coord, Coord, Coord, Coord, Angle, Angle, Coord);
+PinType * CreateNewPin (ElementType *, Coord, Coord, Coord, Coord, Coord, Coord, char *, char *, FlagType);
+PadType * CreateNewPad (ElementType *, Coord, Coord, Coord, Coord, Coord, Coord, Coord, char *, char *, FlagType);
+LineType * CreateNewLineInSymbol (SymbolType *, Coord, Coord, Coord, Coord, Coord);
 void CreateDefaultFont (PCBType *);
-RubberbandType *CreateNewRubberbandEntry (LayerType *, LineType *, PointType *);
-LibraryMenuType *CreateNewNet (LibraryType *, char *, char *);
-LibraryEntryType *CreateNewConnection (LibraryMenuType *, char *);
-AttributeType *CreateNewAttribute (AttributeListType *list, char *name, char *value);
+RubberbandType * CreateNewRubberbandEntry (LayerType *, LineType *, PointType *);
+LibraryMenuType * CreateNewNet (LibraryType *, char *, char *);
+LibraryEntryType * CreateNewConnection (LibraryMenuType *, char *);
+AttributeType * CreateNewAttribute (AttributeListType *list, char *name, char *value);
 
 #endif
